Formula/fish.rb

```diff
@@ -20,12 +20,34 @@ class Fish < Formula
 
   depends_on "pcre2"
 
+  skip_clean "share/fish"
```

Comment:
We shouldn't need this. `skip_clean` applies to inside the Cellar, rather than out of it.